build/pr-variables.yml

```diff
@@ -1,10 +1,10 @@
 variables:
     ResourceGroupRegion: 'westus2'
     resourceGroupRoot: 'msh-fhir-pr'
-    appServicePlanName: '$(resourceGroupRoot)-$(prNumber)-asp'
-    prNumber: $(system.pullRequest.pullRequestNumber)
-    ResourceGroupName: '$(resourceGroupRoot)-$(prNumber)'
+    appServicePlanName: '$(resourceGroupRoot)-$(prName)-asp'    
+    ResourceGroupName: '$(resourceGroupRoot)-$(prName)'
     DeploymentEnvironmentName: 'f$(build.BuildNumber)'
     CrucibleEnvironmentUrl: ''
-    TestEnvironmentName: 'OSS PR$(prNumber)'
+    TestEnvironmentName: 'OSS PR$(prName)'
     ImageTag: '$(build.BuildNumber)'
+    # prName: $(system.pullRequest.pullRequestNumber) # passed in as build variable
```

Comment:
># [](start = 4, length = 1)

YML spec I can find, tells me, it suppose to mean start of comment and it goes all the way to end of the line, but it's probably not the case.

Reply:
I should find a better way to comment this. But wanted to note where this is coming from.